selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/inject/SeleniumTestHandler.java

```diff
@@ -150,6 +155,12 @@ public SeleniumTestHandler() {
   }
 
   private void revokeGithubOauthToken() {
+    // do not revoke if github tests are not being executed
+    if (testGroupsToExclude == null
+        || testGroupsToExclude.contains(TestGroup.GITHUB.toLowerCase())) {
```

Comment:
"toLowerCase()" looks redundant, because "TestGroup.GITHUB" constant is already in lowercase

Reply:
Good catch! I confused constant it with enumeration.